packages/.gitignore

```diff
@@ -0,0 +1 @@
+# place holder to make directory included when we don't have any custom nuget packages
```

Comment:
Keeping this file here, will cause an extra folder in root to be displayed.

Reply:
LOL, that is literally what the comment says it's for that you commented on. We have had custom packages multiple times in the past, if I remove this, I need to remove the custom feed in the nuget.config and remove the folder from being copied in the dockerfile and then if we ever need custom packages again, I'll have to remember to put all those things back.